kafka/upload_new_kafka_version.sh

```diff
@@ -34,14 +39,14 @@ cd "$WORK_DIR" || exit
 bin_file=kafka_2.13-$VERSION.tgz
 src_file=kafka-$VERSION-src.tgz
 
-echo "Downloading Kafka (this can take a while, it is intentionally downloading from a slow mirror that contains all old versions)"
-curl --fail -LOs "https://archive.apache.org/dist/kafka/$VERSION/$bin_file"
-curl --fail -LOs "https://archive.apache.org/dist/kafka/$VERSION/$bin_file.asc"
-curl --fail -LOs "https://archive.apache.org/dist/kafka/$VERSION/$bin_file.sha512"
+echo "Downloading Kafka (if this fails, try switching the BASE_DOWNLOAD_URL to the archive)"
```

Comment:
suggestion: Print two lines (like in the Hive script). One for downloading the binary, one for the source.